sdk/core/core/src/az_span.c

```diff
@@ -432,3 +471,8 @@ AZ_NODISCARD az_result _az_scan_until(az_span self, _az_predicate predicate, int
   }
   return AZ_ERROR_ITEM_NOT_FOUND;
 }
+
+AZ_NODISCARD az_result az_span_append_byte(az_span self, char c, az_span * out)
```

Comment:
Match new function name & signature

Reply:
I take it to be "az_span_append_uint8".

Done.